config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-end
+  get '/guests/:id', to: 'guests#show'
+end
```

Comment:
I'm assuming that this is where you ran out of time and were unable to add the route for the form. But as I mentioned in the comment in that view, you want the route to be `post "/guests/:id/rooms", to: "room_guests#create"` and not be going to `post "/guests/:id", to: "guests#create"` (I'm assuming this route based on the route you had in your form)